rsopt/configuration/setup.py

```diff
@@ -263,6 +266,9 @@ def _check_setup(cls, setup):
                 raise KeyError(f'{key} in setup block for code-type {code} is not recognized.')
         Setup._check_setup(setup)
 
+    def get_sym_link_targets(self):
```

Comment:
This class being named "Python" really is a bit unusual

Naming things after almost keywords, basically. I'm Pretty Sure :tm: this is fine here even in linting rules, as long as we also have setup for other languages (`CeePlusPlus`, anyone?)

Reply:
Perhaps. This follows the naming convention for all the setup classes which mirror the configuration name modulo capitalization.

Reply:
OK